LoopKit/Persistence/Model.xcdatamodeld/Modelv2.xcdatamodel/contents

```diff
@@ -16,9 +18,6 @@
             <uniquenessConstraint>
                 <constraint value="uuid"/>
             </uniquenessConstraint>
-            <uniquenessConstraint>
```

Comment:
Why no longer unique?

Reply:
The `syncIdentifier` is the string (a UUID, as it turns out) that associates all of the CachedCarbObjects as "one user-entered carb entry".  (It is the same thing that HealthKit uses.) So, you could have multiple CachedCarbObjects representing different versions of the same carb entry. They would have different UUIDs, but the same `syncIdentifier` and the `syncVersion` will give you the order they occurred in.